### ISSUE TITLE
Monster initiative does not auto roll when encounter has no players

### ISSUE THREAD

**Issue by azuretower** [cynicaloptimist/improved-initiative#133]
I know this is a small one but maybe someone wishes they could do this.  If I try to start an encounter that has no players then the monsters initiative prompt doesn't appear correctly. The prompt box appears but is empty and once I press enter the encounter starts but every monster has 0 for their initiative.

**Comment by cynicaloptimist** [comment]
What's happening here is that the encounter automatically stops when there are no combatants. Do you think we should change that behavior?

Thanks!

**Comment by azuretower** [comment]
Are combatants players? I do like that the encounter ends when there are no players or monsters left. 

Although I haven’t had a use for a monster only encounter without any players, someone might.  And even if they don’t, it looks kind of bad. It’s not a high priority bug but would be nice to fix.

**Comment by chrisbloom7** [comment]
FWIW: I just came across this tool yesterday and used it behind my DM screen at a tabletop pencil and dice session. Players don't use devices at the table other than for reference, at my request. It was really helpful for me to be able to create some planned and random encounters ahead of time in the app, then pull them up as needed, but I _did not_ use the initiative tracker in the app since we were all rolling at the table for initiative and then using index cards as order trackers. TBH I didn't even start the encounters, I just pulled them up to track XP and stats, then cleared them when it was done. Maybe I'm an edge case, but there was nothing buggy about using it this way, at least not that I noticed. I mention it in case it sways you one way or the other. As a fellow software engineer I'm not sure I would label this a bug so much as an unrealized feature, but as a first time user of the tool it could just be that I don't understand the app enough to know the true intent.
